src/containers/Error/Error.styles.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Stack, styled, Typography } from '@mui/material';
+
+export const StyledMainContainer = styled(Stack)(({ theme }) => ({
+    alignItems: 'center',
+    flex: 1,
+    paddingBottom: theme.spacing(10),
+}));
+
+export const StyledImageContainer = styled(Box)(({ theme }) => ({
+    padding: `${theme.spacing(8)} 0 ${theme.spacing(8)}`,
+    textAlign: 'center',
+
+    [theme.breakpoints.up('md')]: {
+        padding: `${theme.spacing(8)} 0 0`,
+    },
+}));
+
+export const StyledContentContainer = styled(Box)(({ theme }) => ({
+    padding: `0 ${theme.spacing(4)}`,
+}));
+
+export const StyledTitle = styled(Typography)(({ theme }) => ({
+    marginTop: theme.spacing(2),
+
+    [theme.breakpoints.up('md')]: {
+        marginTop: 0,
+    },
+}));
+
+export const StyledText = styled(Typography)(({ theme }) => ({
+    margin: `${theme.spacing(2)} 0 ${theme.spacing(5)}`,
+    color: theme.palette.coolGray['500'],
```

Comment:
As discussed cool gray should be named to grey
```suggestion
    color: theme.palette.grey['500'],
```